tests/lib/fakestore/refresh/snap_asserts.go

```diff
@@ -93,3 +94,40 @@ func NewSnapDeclaration(targetDir string, snap string, headers map[string]interf
 	}
 	return writeAssert(a, targetDir)
 }
+
+// NewRepair signs a repair assertion, including the specified script as the
+// body of the assertion.
+func NewRepair(targetDir string, scriptFilename string, headers map[string]interface{}) (string, error) {
+	db, err := newAssertsDB()
+	if err != nil {
+		return "", err
+	}
+
+	fallbacks := map[string]interface{}{
+		"brand-id":  "testrootorg",
+		"repair-id": "1",
+		"summary":   "some test keys repair",
+	}
+	for k, v := range fallbacks {
+		if _, ok := headers[k]; !ok {
+			headers[k] = v
+		}
+	}
+
+	scriptBodyBytes, err := ioutil.ReadFile(scriptFilename)
+	if err != nil {
+		return "", err
+	}
+
+	headers["authority-id"] = "testrootorg"
+	// note that series is a list for repair assertions
+	headers["series"] = []interface{}{"16"}
+	headers["timestamp"] = time.Now().Format(time.RFC3339)
+
+	a, err := db.Sign(asserts.RepairType, headers, scriptBodyBytes, systestkeys.TestStoreKeyID)
```

Comment:
here we would need to setup to use such key but it shouldn't be too hard

Reply:
yes I refactored this to sign with the new separate root of trust key too